### ISSUE TITLE
Babel's "Unexpected token" error

### ISSUE THREAD

**Issue by alejandroiglesias** [jlengstorf/learn-rollup#45]
Hi, I'm following the Rollup.js Tutorial, Part 2 (How to Bundle Stylesheets and Add LiveReload With Rollup), and after setting postcss, I get an `Unexpected token` error from the babel plugin. If I comment the import line for the CSS file, it goes away and everything goes back to green. Are you aware of this? Am I missing something?

**Comment by alejandroiglesias** [comment]
Nevermind, I found out. It seems the order in which you declare Rollup's plugins array is important. I was putting `babel` before `postcss`. It would be great if you made a note for that in the article, which by the way is great!
Thanks.

**Comment by jlengstorf** [comment]
Good point — I'll add it to my todo list to put a note in there. Thanks!
